docs-source/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -0,0 +1,91 @@
+
+Use the following configuration to specify liveness probe custom script using `livenessProbeCustomScript` domain resource field.
+```
+spec:
+  livenessProbeCustomScript: /u01/customLivenessProbe.sh
+```
+
+Use the following configuration to specify liveness probe custom script using `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable.
+```
+    serverPod:
+      env:
+      - name: LIVENESS_PROBE_CUSTOM_SCRIPT
+        value: /u01/customLivenessProbe.sh
+```
+
+The following environment variables that are populated by the Operator are available for use in the liveness probe custom script which will be invoked by `livenessProbe.sh`.
+
+`ORACLE_HOME` or `MW_HOME`: The Oracle Fusion Middleware software location as a file system path within the container.
+
+`WL_HOME`: The Weblogic Server installation location as a file system path within the container. 
+
+`DOMAIN_HOME`: The domain home location as a file system path within the container.
+
+`JAVA_HOME`: The Java software installation location as a file system path within the container.
+
+`DOMAIN_NAME`: The WebLogic Server domain name.
+
+`DOMAIN_UID`: The domain unique identifier.
+
+`SERVER_NAME`: The WebLogic Server instance name.
+
+`LOG_HOME`: The WebLogic log location as a file system path within the container. This variable is available only if its value is set in the configuration.
+
+**Note**: Additional environment variables that are populated by the Operator and that are not listed above are not supported for use in the liveness probe custom script; their existence or meaning may change in the future.
+
+**Note**: The custom liveness probe script can call `source $DOMAIN_HOME/bin/setDomainEnv.sh` if it needs to setup its PATH or CLASSPATH to access WebLogic utilities in its domain.
+
+**Note**: A custom liveness probe must not fail (exit non-zero) when the WebLogic Server instance itself is unavailable. This could be the case when the WebLogic Server instance is booting or about to boot.
+
+#### Readiness probe customization
+
+The readiness probe is by default configured to use the WebLogic Server ReadyApp framework. The ReadyApp framework allows fine customization of the readiness probe by application's participation in the framework. See [Using the ReadyApp Framework](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/depgd/managing.html#GUID-C98443B1-D368-4CA4-A7A4-97B86FFD3C28) for more details. The readiness probe is used to determine if the server is ready to accept user requests. The readiness is used to determine when a server should be included in a load balancer's endpoints, when a restarted server is fully started in the case of a rolling restart, and for various other purposes.
+
+The readiness probe is by default configured to check readiness every 5 seconds, and to timeout after 5 seconds.  The readiness probe interval and timeout can be customized using the `readinessProbe` attribute under the `serverPod` element of the domain resource.
```

Comment:
The readiness probe is by default configured -> By default, the readiness probe is configured
seconds, and -> seconds and (no comma)
The readiness probe interval and timeout can be customized -> You can customize the readiness probe interval and timeout